packages/client/tests/functional/issues/11740-transaction-stored-query/__snapshots__/tests.ts.snap

```diff
@@ -172,6 +172,30 @@ Invalid \`prisma.resource.create()\` invocation in
 Unique constraint failed on the fields: (\`email\`)
 `;
 
+exports[`issues.11740-transaction-stored-query (provider=sqlite, js_d1) stored query trigger .requestTransaction twice should fail 1`] = `
+
+Invalid \`prisma.resource.create()\` invocation in
+/client/tests/functional/issues/11740-transaction-stored-query/tests.ts:0:0
+
+  XX testIf(engineType !== 'binary' && driverAdapter !== AdapterProviders.JS_PLANETSCALE)(
+  XX   'stored query trigger .requestTransaction twice should fail',
+  XX   async () => {
+→ XX     const query = prisma.resource.create(
+Unique constraint failed on the fields: (\`UNIQUE constraint failed\`)
```

Comment:
This message does not look right, it should list a field after `:`, looks like it has D1 message isntead. Or is it the same for SQLite?

Reply:
If you look a few lines up, it's the same for SQLite indeed
```
exports[`issues.11740-transaction-stored-query (provider=sqlite) stored query triggered twice should fail but not exit process 1`] = `

Invalid \`prisma.resource.create()\` invocation in
/client/tests/functional/issues/11740-transaction-stored-query/tests.ts:0:0

  XX testIf(engineType !== 'binary' && driverAdapter !== AdapterProviders.JS_PLANETSCALE)(
  XX   'stored query triggered twice should fail but not exit process',
  XX   async () => {
→ XX     const query = prisma.resource.create(
Unique constraint failed on the fields: (\`email\`)
`;
```